### ISSUE TITLE
can not add second IMU with xacro:imu_plugin_macro

### ISSUE THREAD

**Issue by maxhofidrone** [ctu-mrs/mrs_simulation#14]
Hello,
I want to simulate a second IMU at the T650. For that I want to use the same macro (in `component_snippets.xacro`) you used for the normal standard IMU (_IMU (ADIS16448)_). Adding an additional sensor to `spawner_params.yaml` and `t650.xacro` works fine. Within `t650.xacro` I just copy paste the existing IMU and change following lines:
```
<xacro:imu_plugin_macro
    namespace="test"
    imu_suffix="new"
    parent_link="base_link"
    imu_topic="/imu_new" 
    .
    .
    .
 </xacro:imu_plugin_macro>
```
The other IMU specific parameters are unchanged for the moment. 
Spawning by an adopted one_drown_baro session (I added the --enable-new-imu)  does not show any errors, but I am never able to see the new _/imu_new_ ROS topic. Furthermore I am also never able to see the original _/imu_ ROS topic nor finding its remapping of the topic name. Can someone please give me some help what I am doing wrong here?

Thank you very much in advance!

Max

**Comment by maxhofidrone** [comment]
Additionally I also added an IMU with a [different IMU plugin](http://gazebosim.org/tutorials?tut=ros_gzplugins#IMU(GazeboRosImu)) which works fine and I am able to see the new topic.  But as it misses the additional IMU parameters of the first example, I don't want to use this example.

**Comment by stibipet** [comment]
Hello Max,

we have also stumbled upon the limitations of the plugin you mentioned. I have recently added a component snippet, which uses a more configurable IMU plugin. You fill find it under "custom_imu_macro" in the `component_snippets.xacro`. Some inspiration on how to set it up may be found in the "vio_macro" block (also in `component_snippets`), which uses the newly added plugin.

```
    <xacro:custom_imu_macro
      namespace="..."
      parent_link="..."
      sensor_name="..."
      update_rate="..."
      topic_name="..."
      noise_mean="..."
      >
      <origin xyz="0 0 0" rpy="0 0 0"/>
    </xacro:custom_imu_macro>
```

Petr
